pandas/tests/base/test_conversion.py

```diff
@@ -173,10 +173,10 @@ def test_iter_box(self):
         # period
         vals = [pd.Period("2011-01-01", freq="M"), pd.Period("2011-01-02", freq="M")]
         s = Series(vals)
-        assert s.dtype == "Period[M]"
+        assert s.dtype == "period[M]"
```

Comment:
likewise

Reply:
Thank you, I rolled my changes here back.